sdk/src/client/node_api/core/routes.rs

```diff
@@ -57,10 +57,11 @@ impl ClientInner {
         url.set_path(PATH);
         let status = crate::client::node_manager::http_client::HttpClient::new(DEFAULT_USER_AGENT.to_string())
             .get(
-                Node {
+                &Node {
                     url,
                     auth: None,
                     disabled: false,
+                    permanode: false,
```

Comment:
permanode doesn't have a health route?

Reply:
Doesn't matter here, we just need to set something